docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -289,13 +289,39 @@ tieredstore:
     low: 0.7
 ```
 
+#### Update Ignore List
```

Comment:
Hmm, this solution does not work if someone is using an existing helm repo? 

After we publish the chart to the stable helm repo, the only command needed to install Alluxio is `helm install alluxio` passing in the config yaml. How does the user choose to not install fuse/job; is there a way to pass in the ignore list to `helm install`?

Reply:
I don't find any way to specify an include/exclude list on a repo so what's there gets installed.

Reply:
Otherwise we can wrap them up in an if condition with properties from config.yaml. 

For the generation of fuse / job sample templates we can commit a `config.yaml` in a sub-dir with those properties enabled and run `helm-generate` on it.